### ISSUE TITLE
`file_size` not reported for `PassportFile`

### ISSUE THREAD

**Issue by Bibo-Joshi** [tdlib/telegram-bot-api#253]
Hi.

I was just testing some passports related features of the `python-telegram-bot` library that I'm maintaining and noticed that the `file_size` parameter of [`PassportFile`](https://core.telegram.org/bots/api#passportfile) was not included in the updates reported by `getUpdates` even though it's marked as required attribute in the documentation.

Here is an anonymized json response I got from `getUpdates`:

<details>

```python
{
    "ok": true,
    "result": [
        {
            "update_id": 219016025,
            "message": {
                "message_id": 28343,
                "from": { ... },
                "chat": { ... },
                "date": 1649595354,
                "passport_data": {
                    "data": [
                        {
                            "type": "passport",
                            "data": "HfLYg...",
                            "front_side": {
                                "file_id": "DgADAgADXRcAAvK4mUrGvDShebTqISME",
                                "file_unique_id": "AwADXRcAAvK4mUo",
                                "file_date": 1649595351
                            },
                            "selfie": {
                                "file_id": "DgADAgADPxcAAnddmUrtny00DHIoLiME",
                                "file_unique_id": "AwADPxcAAnddmUo",
                                "file_date": 1649595351
                            },
                            "hash": "d4B/eUf..."
                        },
                        {
                            "type": "rental_agreement",
                            "files": [
                                {
                                    "file_id": "DgADAgADWxYAAtipmEqYb1oXgLvaxCME",
                                    "file_unique_id": "AwADWxYAAtipmEo",
                                    "file_date": 1649595321
                                }
                            ],
                            "hash": "anZRMln..."
                        },
                        {
                            "type": "phone_number",
                            "phone_number": "123456",
                            "hash": "CVSr..."
                        },
                        {
                            "type": "email",
                            "email": "xxx@yyy.org",
                            "hash": "mKrUZJ..."
                        }
                    ],
                    "credentials": {
                        "data": "Mn4I6...",
                        "hash": "+bD+...",
                        "secret": "ZvQilWVa06..."
                    }
                }
            }
        }
    ]
}
```

</details>

Am I missing something? Or this is a bug in the Bot API/an error in the documentation?

**Comment by Bibo-Joshi** [comment]
Hi. I'd like to kindly follow up on this :)

**Comment by Bibo-Joshi** [comment]
The issue was apparently resolved and the `file_size` is now correctly reported. Closing.

**Comment by levlam** [comment]
This was fixed in Bot API 6.0.2.
